teps/0091-trusted-resources.md

```diff
@@ -364,26 +373,70 @@ type KeylessRef struct {
 
 ### Configuration
 
-Reconciler will respond to verification failure based on ConfigMap settings to:
-1) Return error in reconciler, mark the status as error with `FailedVerification` reason;
-2) Not return error, but mark the TaskRun/PipelineRun as `FailedVerification`;
-3) Skip the verification;
-
-Configuration sample:
-```yaml
+ ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
   name: feature-flag
   namespace: tekton-pipelines
 data:
-  verification-policy: enforce
+  no-match-policy: "deny"
```

Comment:
naming nit: can we use something that explains a bit more context than 'no-match-policy'? ideas:

- 'verification-policy-unmatched'
- 'verification-no-match-policy'

I think I actually like the original name 'verification-policy' a bit better; instead of removing it you could update the options to be something like

- `require-policy` (or `deny-unmatched`): require that a policy be set for all resources (same as no-match-policy: deny)
- `allow-no-policy` (or `allow-unmatched`?): default (same as no-match-policy: allow)

Reply:
Sure! I like the "update the options" approach. Will update this part

Reply:
Updated to use 'verification-policy'  with 

- allow-no-policy
- allow-no-policy-warn
- require-policy

Reply:
To me these new values are more confusing - they're less clear on what the resulting behavior is at a glance.

e.g. `allow-no-policy-warn` could be interpreted as "warn if no policy" or also as "don't allow warn policies". Together you can piece together the differences, but I worry this will be more difficult to determine if looking at only a single value in the config map. I think the `no-match-policy` key better matches the condition we're checking, so I'd prefer to stick with that.

No objections to `verification-policy-unmatched` / `verification-no-match-policy` - can we change back to this with the allow / warn / deny values?

Reply:
ah ok, I will change the naming to "verification-no-match-policy" with allow / warn / deny values!

Reply:
Updated!